code/Model/EditableFormField.php

```diff
@@ -926,19 +913,6 @@ public function getCMSValidator()
             ->setRecord($this);
     }
 
-    /**
-     * Determine effective display rules for this field.
-     *
-     * @return SS_List
-     */
-    public function EffectiveDisplayRules()
```

Comment:
Can't remove public method in a minor release. Needs to be marked as deprecated.

Reply:
It's been restored. I've added a deprecation warning.